packages/core/frontend/sirius-components-core/src/onboarding/NewDocumentArea.tsx

```diff
@@ -0,0 +1,206 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { gql, useMutation } from '@apollo/client';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import IconButton from '@material-ui/core/IconButton';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Snackbar from '@material-ui/core/Snackbar';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { Close as CloseIcon, NoteAdd } from '@material-ui/icons';
+import { useEffect, useState } from 'react';
+import { v4 as uuid } from 'uuid';
+import {
+  GQLCreateDocumentMutationInput,
+  GQLCreateDocumentMutationVariables,
+  GQLInvokeEditingContextActionInput,
+  GQLInvokeEditingContextActionVariables,
+  NewDocumentAreaProps,
+} from './NewDocumentArea.types';
+
+const useNewDocumentAreaStyles = makeStyles(() => ({
+  card: {
+    backgroundColor: 'transparent',
+
+    maxWidth: '330px',
+    minWidth: '330px',
+  },
+  cardContent: {
+    height: '100%',
+    maxHeight: '400px',
```

Comment:
You should rely on spacing instead of hardcoded values and it still feels very odd to have such hardcoded values like `400px`. If you need this component to take 100% of the height of its parent, use a proper configuration of the layout on the parent instead.